Primera documentación readme

3st3r30sc0p10 opened the pull request:
Inicio de la documentación

3st3r30sc0p10 committed:
Primera documentación readme

Inicio de la documentación

1cgonza commented on README.md:
```diff
@@ -4,3 +4,46 @@
+
+Inicia un servidor local para visualizar los datos guardados en MongoDB [http://localhost:8081/](http://localhost:8081/)
+
+## Licencia
```
Esto en que quedó? si podemos sacar las cosas con licencia MIT?

De ser así, toca el archivo de Licencia siguiendo estos pasos: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

3st3r30sc0p10 replied:
Sí, en el contrato de Colev con el IDRC quedó estipulado que lo que se produjera fuera abierto. Entonces podemos poner la licencia que necesitemos

1cgonza replied:
Bien, pero mira el enlace que mande para crear la Licencia en Github.

1cgonza commented on README.md:
```diff
@@ -4,3 +4,46 @@
+# Iniciar contenedores
+docker-compose up
+
+# Instalar docker sin el log
```
No es "instalar" sino "iniciar" contenedor

1cgonza commented on README.md:
```diff
@@ -4,3 +4,46 @@
+
+Inicia un servidor local para ver la aplicación [http://localhost:3000](http://localhost:3000)
+
+Inicia un servidor local para visualizar los datos guardados en MongoDB [http://localhost:8081/](http://localhost:8081/)
```
Poner acá que esto usa Mongo Express para tener una interfaz visual. Útil para desarrollo pero no es necesaria en el despliegue final.

1cgonza reviewed (changes requested):
Algunos comentarios antes de aceptar.

3st3r30sc0p10 committed:
Cambios de palabra, añadir licencia y agregar contenido

se cambió instalar por iniciar. Se añadió información adicional sobre mongo y express y se añadió la licencia MIT al r

1cgonza commented on README.md:
```diff
@@ -11,7 +11,7 @@
 docker-compose up
 
-# Instalar docker sin el log
+# Iniciar contenedores sin el log en terminal
```
Creo que tiene más sentido decir: Inicia servidores sin ocupar el terminal.

1cgonza commented on README.md:
```diff
@@ -42,7 +42,7 @@ npm run dev
 
```
Esto creo que no se debería llamar "Servidores" sino "Aplicaciones" o algo por el estilo pues haces referencia a unos sitios web y no a servidores.

1cgonza commented on README.md:
```diff
@@ -42,7 +42,7 @@ npm run dev
 
 Inicia un servidor local para ver la aplicación [http://localhost:3000](http://localhost:3000)
```
es raro como lo escribes, si inicia un servidor local pero no lo inicia para ver la aplicación. ¿Falta una coma? o ¿son dos frases separadas?

1cgonza commented on README.md:
```diff
@@ -42,7 +42,7 @@ npm run dev
 Inicia un servidor local para ver la aplicación [http://localhost:3000](http://localhost:3000)
 
-Inicia un servidor local para visualizar los datos guardados en MongoDB [http://localhost:8081/](http://localhost:8081/)
+Inicia un servidor local para visualizar los datos guardados en MongoDB [http://localhost:8081/](http://localhost:8081/). La aplicación funciona con MongoDB + Express para tener una interfaz visual. Es útil para el desarrollo de la aplicación, pero no es necesario en el despliegue final.
```
Me pasa lo mismo que con la frase anterior. Inicia el servidor es una cosa y para ver la aplicación es otra cosa.
